webpack.config.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const Dotenv = require('dotenv-webpack'); // eslint-disable-line
```

Comment:
imported dotenv into webpack (so that dotenv does its thing app load I think?)

Reply:
Yep, the earlier the better!